#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_molden.f.dpatch by  <muammar@obey.debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: A patch for changing molden.f

@DPATCH@

--- molden-4.6/molden.f	2008-07-16 13:37:49.000000000 -0430
+++ molden.f	2008-09-01 13:21:47.000000000 -0430
@@ -275,7 +275,7 @@
 c     with some versions of gfortran it helps to uncomment this line:
 c      external iargc
 c GCC 4.0 or higher: 
-c      external gfortran_iargc
+      external gfortran_iargc
       dimension fcnt(mxcntr)
       character title*80, keywrd*160, keyhlp*80, keyori*160
       character tmpstr*80,movfil*80,povfil*80,oglfil*80
